### DISCUSSION TITLE
a/deploying-stable-diffusion-webui

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#5] [Announcements]
# a/deploying-stable-diffusion-webui

https://blog.yasking.org/a/deploying-stable-diffusion-webui.html

**Comment by wuankangg** [comment]
你好，我想请教一下我按照您的步骤pip3 install -r requirements.txt -i https://pypi.tuna.tsinghua.edu.cn/simple 通过requirement.txt文件下载依赖项的时候报错“ERROR: Exception:
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3108, in _dep_map
    return self.__dep_map
  File "/home/ubuntu/.local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2901, in __getattr__
    raise AttributeError(attr)
AttributeError: _DistInfoDistribution__dep_map
”，请问这怎么解决

**Reply by sincerefly** [reply]
```bash
pip3 install -r requirements.txt -i https://pypi.tuna.tsinghua.edu.cn/simple
```

这个命令指定了使用清华源下载，尝试不带 `-i` 参数安装试试 `pip3 install -r requirements.txt`

或者使用豆瓣源试下

```bash
pip3 install -r requirements.txt -i https://pypi.douban.com/simple
```

**Comment by wuankangg** [comment]
pkg_resources.extern.packaging._tokenizer.ParserSyntaxError: Expected closing RIGHT_PARENTHESIS
    torch (>=1.9.*)会报这种异常
          ~~~~~~^

**Reply by sincerefly** [reply]
这个错误你是不是手动修改  requirements.txt 文件了，感觉有语法错误，在执行什么命令时候遇到的呢 🤔
